### ISSUE TITLE
Remove default sites

### ISSUE THREAD

**Issue by Mte90** [topdown/VVV-Dashboard#59]
With VVV 2.0 there are anymore default sites because you can remove them so in defaults.php on line 82 I cleaned the array to remove them.

**Comment by Mte90** [comment]
I removed also all the reference in the code

**Comment by topdown** [comment]
I just tried installing VVV2 the other day and it had/installed the wordpress-default and wordpress-develop

**Comment by Mte90** [comment]
Yes but you can remove them very easily so is not so much useful to have in the list of websites if you don't need them. I want to have the list only of website that are installed.

**Comment by tomjn** [comment]
You can comment out or delete their entries in `vvv-custom.yml` and reprovision, and they'll be gone

**Comment by Mte90** [comment]
Yes today with VVV 2.x is not required anymore but this dashboard have some website referenced in the code.
